README.md

```diff
@@ -41,13 +41,75 @@ to run remote actions in a recent Ubuntu container.
 
 ## Configuration
 
+### Data and log storage
+
 EngFlow stores its caches and logs in the location pointed to by the `$DATA_DIR`
 environmental variable. Mount this location onto the host to preserve data
 across container invocations and view the service logs. If remote execution is
 used, `$DATA_DIR` must have the same path on the host and in the
 container. Additionally, the Docker Unix socket must be mounted into the
 container to allow EngFlow to execute actions in Docker containers.
 
+### Setting up .bazelrc and using a custom Docker image
+
+If you wish to use a custom image for a specific workspace add the following to
+the **WORKSPACE** file:
+```
+git_repository(
```

Comment:
Consider including the `load` to get `git_repository` to make the copy/pasta easier

Reply:
Done.